data/XML/mounts.xml

```diff
@@ -16,31 +16,31 @@
 	<mount id="14" clientid="388" name="Tiger Slug" speed="20" premium="yes" />
 	<mount id="15" clientid="389" name="Uniwheel" speed="20" premium="yes" />
 	<mount id="16" clientid="390" name="Crystal Wolf" speed="20" premium="yes" />
-	<mount id="17" clientid="392" name="Brown War Horse" speed="20" premium="yes" />
+	<mount id="17" clientid="392" name="War Horse" speed="20" premium="yes" />
 	<mount id="18" clientid="401" name="Kingly Deer" speed="20" premium="yes" />
 	<mount id="19" clientid="402" name="Tamed Panda" speed="20" premium="yes" />
 	<mount id="20" clientid="405" name="Dromedary" speed="20" premium="yes" />
-	<mount id="21" clientid="406" name="King Scorpion" speed="20" premium="yes" />
+	<mount id="21" clientid="406" name="Scorpion King" speed="20" premium="yes" />
 	<mount id="22" clientid="421" name="Rented Horse" speed="20" premium="no" />
 	<mount id="23" clientid="426" name="Armoured War Horse" speed="20" premium="yes" />
 	<mount id="24" clientid="427" name="Shadow Draptor" speed="20" premium="yes" />
 	<mount id="25" clientid="437" name="Rented Horse" speed="20" premium="no" />
 	<mount id="26" clientid="438" name="Rented Horse" speed="20" premium="no" />
-	<mount id="27" clientid="447" name="Ladybug" speed="20" premium="yes" />
-	<mount id="28" clientid="450" name="Manta" speed="20" premium="yes" />
+	<mount id="27" clientid="447" name="Lady Bug" speed="20" premium="yes" />
```

Comment:
I used some videos to see the names. I've seen "Lady Bug" in:
https://youtu.be/-MG3OaMyr-4?t=11m8s
https://youtu.be/drDK51XIt38?t=4m56s